### PR TITLE
Add container tests

### PR DESCRIPTION
**Pull request by mongrelion**
After building the Docker image that is going to be pushed to the Hub, we're going to run a simple container test to make sure that it actually is "working" (as in, we can curl the main endpoint of the app inside the container)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,4 +25,5 @@ after_success:
   - if [ -z "$DOCKER_PASS" ]; then echo "Build triggered by external PR. Skipping docker push" && exit 0; fi
   - docker login -u $DOCKER_USER -p $DOCKER_PASS;
   - ./scripts/build.sh
+  - ./scripts/smokey.sh
```
Comment (philwinder) [inline .travis.yml]: This is a container level test. Not a smoke test. Rename to `container-test.sh` to be consistent with other repos.

File: scripts/smokey.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+REPO=${GROUP}/$(basename front-end);
+
+CID=$($DOCKER_CMD run -d --name smoketest -p 8080:8079 ${REPO}:${COMMIT})
+
+for i in 1 2 3 4 5
+do
+  curl -s --head http://localhost:8080/ > /dev/null
+  if [ $? -eq "0" ]
+  then
+    echo "smoke test passed"
```
Comment (philwinder) [inline scripts/smokey.sh]: And here.

File: scripts/test-container.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```
Comment (alex-glv) [inline scripts/test-container.sh]: I think this file belongs to test/

**Comment by mongrelion** [comment]
Merging this as I have already addressed every comment on the PR and it's looking good and working.

**Pull request merged**